libmikmod/playercode/mplayer.c

```diff
@@ -50,6 +50,10 @@ extern long int random(void);
 /* The currently playing module */
 MODULE *pf = NULL;
 
+/* Different playing information */
+UBYTE farcurtempo;   /* Farandole current speed */
+SWORD fartempobend;  /* Used by the Farandole fine tempo effects and store the current bend value */
+
```

Comment:
I agree with sezero that fewer globals is better in a library like MikMod—this is already a pretty big problem in this library and prevents the playback of multiple modules simultaneously when using a software mixing driver (yes, there are actual use cases for that). I don't know that there's a better private module area to place these in than `MP_CONTROL` like sezero recommended.